src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.cs

```diff
@@ -1326,6 +2229,16 @@ public abstract partial class SHA512 : System.Security.Cryptography.HashAlgorith
         public static System.Threading.Tasks.ValueTask<byte[]> HashDataAsync(System.IO.Stream source, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public static bool TryHashData(System.ReadOnlySpan<byte> source, System.Span<byte> destination, out int bytesWritten) { throw null; }
     }
+    public sealed partial class SHA512CryptoServiceProvider : System.Security.Cryptography.SHA512
```

Comment:
This appears to be missing the EBN and Obsolete attribute that the other CryptoServiceProviders have.

```
    [System.ObsoleteAttribute("Derived cryptographic types are obsolete. Use the Create method on the base type instead.", DiagnosticId = "SYSLIB0021", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
    [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
    ```

Reply:
Weird.  They're there in the impl.